lib/server/blog.js

```diff
@@ -34,8 +34,12 @@ function fileToUrl(file) {
 
 function getPagesMarkup(numOfBlog, config) {
   const BlogPageLayout = require('../core/BlogPageLayout.js');
+  const env = require('./env');
   const blogPages = {};
   const perPage = 10;
+
+  env.translation.setLanguage('en');
```

Comment:
But there are translations in `<Site>` component. I think it is safer to explicitly set language to default `en`, because it may happen, that current translation language will not be 'en' due to some doc page rendered previously using other language.